cgmml/data_utils/dataset_generation/train_test_split.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import random
+
+import dbutils
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--db_connection_file', default="dbconnection.json", help='json file with secrets to connect to database')  # noqa: E501
```

Comment:
Instead of ignoring the error that the line is too long with putting `  # noqa: E501`, you should actually split the line so each line is less than 120 chars long
```suggestion
    parser.add_argument('--db_connection_file', default="dbconnection.json",
                        help='json file with secrets to connect to database')
```